cypress/support/api/apiTests.spec.js

```diff
@@ -0,0 +1,77 @@
+import {DELETE, GET, POST, PUT} from "cypress/support/api/apiUtils.js";
+import {
+    createDashboard,
+    deleteDashboardById,
+    getActivityItemId,
+    updateDashboardById
+} from "cypress/support/api/endpoints.js";
+
+describe('CRUD Tests', () => {
+    it('should retrieve activity item by ID', () => {
+
+        const projectName = 'DEMO DASHBOARD';
+        const itemId = 10;
+
+        // Make the GET request using the utility function
+        GET(getActivityItemId(projectName, itemId)).then(response => {
+            // Assert that the response status code is 200
+            expect(response.status).to.eq(200);
+        });
+    });
+
+    it('should delete dashboard by ID', () => {
+        // Assuming you have a valid projectId and dashboardId
+        const projectName = 'DEMO DASHBOARD';
```

Comment:
It'd be easier if you had a class Dashboard. In that case, you could operate with a dashboard object. I mean smth like that:
```
const dashboardToDelete = new Dashboard('DEMO DASHBOARD', 'DashboardId');
const response = await DELETE(deleteDashboardById(dashboardToDelete));
```